virt/install/virt-specifying-nodes-for-virtualization-components.adoc

```diff
@@ -0,0 +1,29 @@
+[id="virt-specifying-nodes-for-virtualization-components"]
+= Specifying nodes for {VirtProductName} components
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-specifying-nodes-for-virtualization-components
+toc::[]
+
+Specify the nodes where you want {VirtProductName} to deploy Operators, workloads, and controllers by configuring `nodePlacement` rules during xref:../../virt/install/installing-virt-cli.adoc#virt-deploying-operator-cli_installing-virt-cli[installation].
```

Comment:
Here we completely miss a section:
HCO CR controls where HCO (and other {VirtProductName} operators) deploy their operands.
But all the {VirtProductName} operators are instead deployed by the Operator Lifecycle Manager so the placement of the operators can only be influenced acting at OLM level modifying the OLM subscription (an operator cannot influence its own placement when already there, we can only act one level above!)

See also:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/docs/cluster-configuration.md#operators-placement

And
https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md#nodeselector
https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md#tolerations
for OLM examples.

Please notice that OLM supports only nodeSelectors and tolerations but not affinity.

Reply:
I will add this section ASAP; it was not present in the pull request that I used as my reference (https://github.com/kubevirt/hyperconverged-cluster-operator/pull/842/files). Thank you for pointing it out, @tiraboschi